packages/perseus/src/widgets/__stories__/number-line.stories.tsx

```diff
@@ -12,3 +18,42 @@ type StoryArgs = Record<any, any>;
 export const Question1 = (args: StoryArgs): React.ReactElement => {
     return <RendererWithDebugUI question={question1} />;
 };
+
```

Comment:
I wanted to add a story that shows this particular widget form as there's currently no where else to test the number-line with the tick controller!